spec/controllers/content_item_signups_controller_spec.rb

```diff
@@ -48,6 +48,16 @@
       expect(response).to have_http_status(:bad_request)
     end
 
+    it "returns a 400 when the user content store returns a bad request response" do
```

Comment:
I assume adding 'user' here is a mistake? So just `"returns a 400 when the content store returns a bad request response"`

Reply:
Thanks, good spot.